spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -300,6 +487,18 @@ function onAcknowledgePacket(
 function onTimeoutPacket(packet: Packet) {
   // the packet timed-out, so refund the tokens
   refundTokens(packet)
+  // if this packet was forwarded from a previously received packet
+  // then we need to send notification of timeout in reverse
+  reversePacket = store.get(packetForwardPath(packet.sourcePort, packet.sourceChannel, packet.Sequence))
+  if reversePacket != nil {
+    ack = FungibleTokenPacketAcknowledgement{false, "forwarding packet timed out"}
+    // prefix with forwardingPath so we know which chain it timed out on
+    ack.error = prefix(ack.error, reversePacket.sourcePort + "/" + reversePacket.sourceChannel)
```

Comment:
Just for my understanding: if packet has to go chain A -> chain B -> chain C, the packet stored in `packetForwardPath` will be the packet sent from chain A to chain B, isn't it? Then in a potential time out between chain B and chain C, the `reversePacket.sourcePort` will be source port on chain A and `reversePacket.sourceChannel` will be source channel on chain A? Is that really what we want here?